internal/rpc/rpc.go

```diff
@@ -118,12 +120,29 @@ func (rpc *Client) checkSupportedMethods() error {
 	}
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +115]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->this function is getting longer and longer, maybe time to break it down to smaller functions?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/insight/102?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
done